PubHub/PubHub.API/Domain/Entities/User.cs

```diff
@@ -11,8 +11,8 @@ public sealed class User
         public required DateOnly Birthday { get; set; }
 
         #region Navs
-        public Account Account { get; set; } = null!;
-        public IList<UserBook> UserBooks { get; set; }
+        public Account? Account { get; set; }
```

Comment:
Same as [this comment](https://github.com/Mike-Mortensen-Portfolio/PubHub_H6_Final/pull/65/files#r1542453037).